django_rq/templates/django_rq/job_detail.html

```diff
@@ -186,6 +186,12 @@
 
     <div class="submit-row">
         <p class="deletelink-box"><a href="delete/" class="deletelink">Delete</a></p>
+        {% if job.is_started %}
+            <form method = 'POST' action = "{% url 'rq_requeue_job' queue_index job.id %}">
```

Comment:
This does not look like it points to the correct URL.

Reply:
@gabriels1234 let me know if you can fix this, I'd like to merge this in time for the next release :)

Reply:
Done.